<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceWidget</class>
 <widget class="QWidget" name="ServiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServiceWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_auto_start">
       <property name="text">
        <string>一键启动</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_auto_stop">
       <property name="text">
        <string>一键停止</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTableWidget" name="tb_services">
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_checked_stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>选中停止</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_checked_start">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>选中启动</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
